lib/honeydew.ex

```diff
@@ -179,6 +179,17 @@ defmodule Honeydew do
     jobs
   end
 
+  @doc """
+  Cancels a job.
+
+  The return value depends on the status of the job.
+
+  * `:ok` - Job had not been started and was able to be cancelled.
+  * `{:error, :in_progress}` - Job was in progress and unable to be cancelled.
+  * `nil` - Job was not found on the queue (or already processed) and was unable
```

Comment:
hm, do you suppose that this should be `{:error, :not_found}`, so the user doesn't have to match on two different things to find out if the job was cancelled?

Reply:
That makes more sense to me. 👍  I'll update the code and tests.